applications/operators/olm-creating-policy.adoc

```diff
@@ -0,0 +1,39 @@
+[id="olm-creating-policy"]
+= Creating policy for Operator installations and upgrades
+include::modules/common-attributes.adoc[]
+:context: olm-creating-policy
+
+toc::[]
+
+Operators can require wide privileges to run, and the required privileges can
+change between versions. Operator Lifecycle Manager (OLM) runs with
+`cluster-admin` privileges. By default, Operator authors can specify any set of
+permissions in the ClusterServiceVersion (CSV) and OLM will consequently grant
+it to the Operator.
+
+Cluster administrators should take measures to ensure that an Operator cannot
+achieve cluster-scoped privileges and that users cannot escalate privileges
+using OLM. One method for locking this down requires cluster administrators
+auditing Operators before they are added to the cluster. Starting in
+{product-title} 4.2, however, administrators are provided tools for determining
```

Comment:
Do we need the version number here since this is going into 4.2 only? 
_requires cluster administrators auditing Operators before they are added to the cluster using tool provided for determining_

Reply:
Removed the 4.2 mention and will let Release Notes make the distinction.